pom.xml

```diff
@@ -107,6 +107,7 @@
     <module>blueocean-config</module>
     <module>blueocean</module>
     <module>blueocean-jwt</module>
+    <module>blueocean-docker-dev-image</module>
```

Comment:
so for folks building blue ocean on their desktop - this won't add significant overhead?